frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -318,10 +370,26 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     );
   };
 
+  if (_.isEmpty(filteredItems)) {
+    return (
+      <>
+        <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
+          <Title headingLevel="h5" size="lg">
+            No operators available
+          </Title>
+          <EmptyStateBody>
+            There are no operators that match your operating system ({window.SERVER_FLAGS.GOOS}) 
+            and architecture ({window.SERVER_FLAGS.GOARCH}).
```

Comment:
```tsx
{window.SERVER_FLAGS.GOOS && window.SERVER_FLAGS.GOARCH && (
  <EmptyStateBody>
    There are no operators that match operating system {window.SERVER_FLAGS.GOOS} 
    and architecture {window.SERVER_FLAGS.GOARCH}.
  </EmptyStateBody>
)}

Reply:
Yeah that makes sense.  So:

```
No operators available
```
If we don't have `arch` and `os` but for some reason nothing in the list.

```
No operators available

There are no operators that match operating system FooOS and 
              architecture BarARCH}.
```
if we have these values.